src/views/Game/Game.tsx

```diff
@@ -449,6 +449,11 @@ export class Game extends React.PureComponent<GameProperties, any> {
             this.goban.setMode("analyze");
         }
 
+        // We need an initial score for the first display rendering (which is not set in the constructor).
+        // Best to get this from the engine, so we know we have the right structure...
+        this.state.score = this.goban.engine.computeScore(true);
```

Comment:
Nice find! There's a react-ism we need to account for, which is we can never set the state like this *except* for in the constructor. Everywhere else, we use `this.setState`, specifically in this case it would be `this.setState({score: this.goban.engine.computeScore(true)});`